build/Dockerfile.build

```diff
@@ -10,32 +10,33 @@ ENV ADBLOCK true
 ENV DISABLE_OPENCOLLECTIVE true
 ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD true
 
-FROM offen_node as auditorium
+FROM offen_node as script
 ARG skip_locales
 ENV SKIP_LOCALES=$skip_locales
-COPY ./auditorium/package.json ./auditorium/package-lock.json /code/deps/
+COPY ./script/package.json ./script/package-lock.json /code/deps/
 WORKDIR /code/deps
 RUN npm ci
-COPY ./auditorium /code/auditorium
+COPY ./script /code/script
 COPY ./banner.txt /code/banner.txt
-COPY ./locales /code/auditorium/locales
-WORKDIR /code/auditorium
-RUN cp -a /code/deps/node_modules /code/auditorium/
+COPY ./locales /code/script/locales
+WORKDIR /code/script
+RUN cp -a /code/deps/node_modules /code/script/
 ENV NODE_ENV production
 RUN npm run build
 RUN npm run licenses
 
-FROM offen_node as script
+FROM offen_node as auditorium
 ARG skip_locales
 ENV SKIP_LOCALES=$skip_locales
-COPY ./script/package.json ./script/package-lock.json /code/deps/
+COPY ./auditorium/package.json ./auditorium/package-lock.json /code/deps/
 WORKDIR /code/deps
 RUN npm ci
-COPY ./script /code/script
+COPY ./auditorium /code/auditorium
+COPY --from=script ./code/script/index.js /code/auditorium/script/index.js
```

Comment:
This copies the entrypoint file when instead we need to copy the bundled versions. See my comment on how to do this.

Reply:
Shouldn't this invoke the behavior we need?

`COPY --from=script /code/script/dist/index.js /script/script.js`

Reply:
There is a bundle created per locale (the script logs user facing messages in some cases which are localized) so we need t copy the entire dist directory and then pick the correct one in the build for `auditorium`.

See: https://github.com/offen/offen/blob/c7aed87b2746766346b6908434c6e9d4ae2c6df3/script/gulpfile.js#L43-L44 and https://github.com/offen/offen/blob/c7aed87b2746766346b6908434c6e9d4ae2c6df3/script/gulpfile.js#L76-L84

Reply:
To not bother you with more questions I'll need some time to work me through the build process. To be honest it is a lot magic for me right now.

I'm not quite sure how gulp works together with the `dockerfile.build` furthermore my overall gulp knowledge is very limited.

Reply:
If you have questions while doing so, don't worry about bothering me. In any case, take your time. Thanks.